src/rustdoc_schema.graphql

```diff
@@ -429,6 +429,15 @@ interface Variant implements Item {
 
   # own edges
   field: [StructField!]
+  discriminant: Discriminant
+}
+
+"""
+https://docs.rs/rustdoc-types/latest/rustdoc_types/struct.Discriminant.html
+"""
+type Discriminant {
+  expr: String!
+  value: String!
```

Comment:
Could you document what each of these hold? Some of the less-obvious properties have docs, and these seem pretty non-obvious — especially given that they are both `String!`-typed, which seems unexpected to me.

Reply:
I explain in a comment below why rustdoc stores both, especially `value` as strings.